src/invstorage.h

```diff
@@ -116,8 +117,8 @@ class INvStorage {
   virtual bool loadEcuRegistered() = 0;
   virtual void clearEcuRegistered() = 0;
 
-  virtual void storeInstalledVersions(const std::map<std::string, std::string>& installed_versions) = 0;
-  virtual bool loadInstalledVersions(std::map<std::string, std::string>* installed_versions) = 0;
+  virtual void storeInstalledVersions(const Json::Value& installed_versions) = 0;
+  virtual bool loadInstalledVersions(Json::Value* installed_versions) = 0;
```

Comment:
What was the reason for having an untyped value here?  Would a real class be a better/safer option?

Reply:
After I added third coulumn, we cannot use pair anymore because it holds only two values. So we need or create some struct and serealize/deserealize from json or just use json avoid those conversions. So I choosed second option. But if You insist I could create structure.